src/lib/ptree/ptree.lua

```diff
@@ -392,7 +372,7 @@ function Manager:sample_active_counters()
 end
 
 function Manager:start_worker_for_graph(id, graph)
-   local scheduling = self:compute_scheduling_for_worker(id, graph)
+   local scheduling = self.worker_default_scheduling
```

Comment:
This would put all workers on the same CPU, no?  Again: what bug is being fixed here?

Reply:
The PR fixes issue #958, which to my interpretation means to remove the hack of groveling through the list or command line arguments to determine the pci address used by the lwAFTR. Instead what's suggested in the description of the issue is to fetch pci address from lwAFTR's configuration.

I thought when you asked earlier this question it was a rhetoric question since the problem with groveling the command line args in `ptree.lua` was that the proposed PR didn't fix the main issue underneath, which was that from my point of view that `ptree.lua` was slightly coupled to the lwAFTR. Now by fetching the pci addresses in `program/lwaftr/setup.lua` this coupling is gone.